src/components/App.vue

```diff
@@ -125,11 +125,13 @@
                         <v-icon size="64">mdi-arrow-down-bold</v-icon>
                       </div>
                       <div>Drag &amp; drop your DICOM files.</div>
+                      <br />
+                      <br />
+                      <br />
```

Comment:
I would suggest changing the below `class="mt-16"` to `class="offset-margin"`. At the bottom of this file inside the `<style scoped>` tag, add the following lines:

```css
.offset-margin {
  margin-top: 128px;
}
```

Reply:
Done.